app/2.1.x/hybrid-mode.md

```diff
@@ -27,6 +27,24 @@ Deploying using Hybrid mode has a number of benefits:
 * Easiness of management, since an admin will only need to interact with the CP nodes to control
   and monitor the status of the entire Kong cluster.
 
+## Configuration Properties
+
+Hybrid Mode introduces the following configuration properties:
+
+Parameter | Description
+--- | ---
+`cluter_listen` *Optional* | List of addresses and ports on which the Control Plane will listen for incoming Data Plane connections. Defaults to `0.0.0.0:8005`.
+`cluster_control_plane` *Required* | Address and port that the Data Plane nodes use to connect to the Control Plane. Must point to the port configured using the `cluster_listen` property on the Control Plane node.
+`cluster_mtls` *Optional* | One of `"shared"` or `"pki"`. Indicates whether Hybrid Mode will use a shared certificate/key pair for CP/DP mTLS or if PKI mode will be used.  Defaults to `"shared"`.
+
+The following properties are used differently between "shared" and "PKI" modes:
+ 
+Parameter | Description | Shared Mode | PKI Mode
+--- | --- | --- | ---
+`cluster_cert` and `cluster_cert_key` *Required* | Certificate/key pair used for mTLS between CP/DP nodes. | Same between CP/DP nodes. | Unique certificate for each node, generated from the CA specified by `cluster_ca_cert`.
+`cluster_ca_cert` *Required in PKI mode* | The trusted CA certificate file in PEM format used to verify the `cluster_cert`. | *Ignored* | CA certificate used to verify `cluster_cert`, same between CP/DP nodes. *Required*
+`cluster_server_name` *Required in PKI mode* | The SNI Server Name presented by the DP node mTLS handshake. | *Ignored* (uses `"kong_clustering"`) | The CP node will verify that it matches the Common Name (CN) or Subject Alternative Name (SAN) of the node certificate in `cluster_ca_cert`. *Required for DP nodes.*
```

Comment:
```suggestion
`cluster_server_name` *Required in PKI mode* | The SNI Server Name presented by the DP node mTLS handshake. | *Ignored* (uses `"kong_clustering"`) | The DP node will verify that the certificate presented by CP matches the Common Name (CN) or Subject Alternative Name (SAN) of the configured `cluster_ca_cert`. *Required for DP nodes.*
```

Reply:
so i had it backwards.... what about the last part "required by XXX nodes"  is it DP or CP?

Reply:
I missed it completely, that should do it.

We should still change the wording in the middle though, it's the DP node using this information, CP ignores it.